kafka/src/main/java/com/linecorp/armeria/server/logging/structured/kafka/KafkaStructuredLoggingService.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.logging.structured.kafka;
+
+import java.util.Properties;
+import java.util.function.Function;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+
+import com.linecorp.armeria.common.Request;
+import com.linecorp.armeria.common.Response;
+import com.linecorp.armeria.server.Service;
+import com.linecorp.armeria.server.logging.structured.ApacheThriftStructuredLog;
+import com.linecorp.armeria.server.logging.structured.StructuredLogBuilder;
+import com.linecorp.armeria.server.logging.structured.StructuredLogWriter;
+import com.linecorp.armeria.server.logging.structured.StructuredLoggingService;
+
+/**
+ * Provides handy methods to create {@link StructuredLoggingService} with integration to Kafka.
+ */
+public final class KafkaStructuredLoggingService {
+
+    private static final Properties defaultProducerConfig = new Properties();
+
+    static {
+        defaultProducerConfig.setProperty(ProducerConfig.CLIENT_ID_CONFIG,
+                                          KafkaStructuredLoggingService.class.getSimpleName());
+        defaultProducerConfig.setProperty(ProducerConfig.ACKS_CONFIG, "all");
+        defaultProducerConfig.setProperty(ProducerConfig.RETRIES_CONFIG, "3");
+    }
+
+    private KafkaStructuredLoggingService() {}
+
+    private static Properties newDefaultConfig(String bootstrapServers) {
+        Properties producerConfig = new Properties();
+        for (String key : defaultProducerConfig.stringPropertyNames()) {
+            producerConfig.setProperty(key, defaultProducerConfig.getProperty(key));
```

Comment:
Maybe we can just set the default properties all here and get rid of `defaultProducerConfig`?